code/modules/mob/living/carbon/human/species/moth.dm

```diff
@@ -0,0 +1,217 @@
+#define COCOON_WEAVE_DELAY 5 SECONDS
+#define COCOON_EMERGE_DELAY 15 SECONDS
+#define COCOON_HARM_AMOUNT 50
+#define COCOON_NUTRITION_AMOUNT -200
+#define FLYSWATTER_DAMAGE_MULTIPLIER 9
+
+/datum/species/moth
+	name = "Nian"
+	name_plural = "Nianae"
+	language = "Tkachi"
+	icobase = 'icons/mob/human_races/r_moth.dmi'
+	inherent_factions = list("nian")
+	species_traits = list(NO_HAIR)
+	inherent_biotypes = MOB_ORGANIC | MOB_HUMANOID | MOB_BUG
+	clothing_flags = HAS_UNDERWEAR | HAS_UNDERSHIRT
+	bodyflags = HAS_HEAD_ACCESSORY | HAS_HEAD_MARKINGS | HAS_BODY_MARKINGS | HAS_WING | BALD
+	reagent_tag = PROCESS_ORG
+	dietflags = DIET_HERB
+	tox_mod = 1.5
+	blood_color = "#b9ae9c"
+	unarmed_type = /datum/unarmed_attack/claws
+	scream_verb = "buzzes"
+	male_scream_sound = 'sound/voice/scream_moth.ogg'
+	female_scream_sound = 'sound/voice/scream_moth.ogg'
+	default_headacc = "Plain Antennae"
+	default_bodyacc = "Plain Wings"
+	wing = "plain"
+	eyes = "moth_eyes_s"
+	butt_sprite = "nian"
+	siemens_coeff = 1.5
+
+	has_organ = list(
+		"heart" =    /obj/item/organ/internal/heart/nian,
+		"lungs" =    /obj/item/organ/internal/lungs/nian,
+		"liver" =    /obj/item/organ/internal/liver/nian,
+		"kidneys" =  /obj/item/organ/internal/kidneys/nian,
+		"brain" =    /obj/item/organ/internal/brain/nian,
+		"eyes" =     /obj/item/organ/internal/eyes/nian
+		)
+
+	optional_body_accessory = FALSE
+
+	var/datum/action/innate/cocoon/cocoon
+
+	suicide_messages = list(
+		"is attempting to nibble their antenna off!",
+		"is jamming their legs into their eye sockets!",
+		"is twisting their own neck!",
+		"is cracking their exoskeleton!",
+		"is ripping their wings off!",
+		"is holding their breath!")
```

Comment:
```suggestion
		"is holding their breath!"
	)
```

How do you manage 2 styles of brackets literally 4 statements apart?

Reply:
yes